tequilapi/endpoints/sse_handler.go

```diff
@@ -222,6 +224,15 @@ type consumerConnectionRes struct {
 }
 
 func mapState(event stateEvent.State) stateRes {
+	identitiesRes := make([]contract.IdentityDTO, len(event.Identities))
+	for idx, identity := range event.Identities {
+		identityRes := contract.IdentityDTO{
```

Comment:
confusing place - `contract` package with single model

Reply:
This is different PR: https://github.com/mysteriumnetwork/node/pull/1906
I've extracted the contract to a separate package for re-use between the server and the client. Not refactoring all at once, but rather following the boy scout rule. What is confusing?